tests/Testing/Concerns/ElementAssertionsTest.php

```diff
@@ -13,50 +16,85 @@ class ElementAssertionsTest extends Framework_Test_Case {
 		<section>Example Section</section>
 		<div class="test-class">Example Div By Class</div>
 		<div id="test-id">Example Div By ID</div>
+		<ul>
+			<li>Item 1</li>
+			<li>Item 2</li>
+			<li data-testid="test-item">Item 3</li>
+		</ul>
 	</div>';
 
-	public function test_element_exists_by_xpath() {
-		$response = new Test_Response( $this->test_content );
+	protected function response(): Test_Response {
+		return new Test_Response( $this->test_content );
+	}
 
-		$response->assertElementExists( '//div' );
-		$response->assertElementExists( '//section' );
-		$response->assertElementMissing( '//article' );
+	public function test_element_exists_by_xpath() {
+		$this->response()
+			->assertElementExists( '//div' )
+			->assertElementExists( '//section' )
+			->assertElementMissing( '//article' );
 	}
 
 	public function test_element_exists_by_id() {
-		$response = new Test_Response( $this->test_content );
-
-		$response->assertElementExistsById( 'test-id' );
-		$response->assertElementExistsById( '#test-id' );
-		$response->assertElementMissingById( 'missing-id' );
-		$response->assertElementMissingById( '.missing-id' );
+		$this->response()
+			->assertElementExistsById( 'test-id' )
+			->assertElementExistsById( '#test-id' )
+			->assertElementMissingById( 'missing-id' )
+			->assertElementMissingById( '.missing-id' );
```

Comment:
I think this should really be `#missing-id`, no?

Reply:
`missing-id` really!